Jellyfin.Api/Enums/UserRole.cs

```diff
@@ -0,0 +1,23 @@
+namespace Jellyfin.Api.Enums
+{
+    /// <summary>
+    /// Enum for user roles used in the authentication and authorization for the API.
+    /// </summary>
+    public enum UserRole
```

Comment:
Since this is only user for the names, isn't a static class with const strings a better idea?

Reply:
Yes